### PR TITLE
Update layout of landingpage

### PR DESCRIPTION
**Pull request by bramkragten**
CleanShot 2023-09-04 at 14 35 02

On error:
CleanShot 2023-09-11 at 15 45 55@2x

### CODE REVIEW

File: rootfs/usr/share/www/static/styles.css [hunk 1]
```diff
@@ -4,160 +4,391 @@
...
+dialog button svg {
+  height: 100%;
+  width: 100%;
+}
+
+.headline {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 24px;
```
Comment (piitaya) [inline rootfs/usr/share/www/static/styles.css:313]:
```suggestion
  padding: 24px 24px 0px;
```

File: rootfs/usr/share/www/index.html [hunk 2]
```diff
@@ -1,169 +1,149 @@
...
+    <a href="https://www.home-assistant.io/newsletter/" target="_blank" rel="noreferrer noopener" class="link">
+      <div>
+        <img loading="lazy" src="/landingpage/static/logo_twitter.png" />
+        Twitter
+      </div>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+          d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
+      </svg>
+    </a>
```
Comment (piitaya) [inline rootfs/usr/share/www/index.html:118]: Can we add top and bottom padding (8px) to match the onboarding style?
Also, the link are all the same.

File: rootfs/usr/share/www/static/styles.css [hunk 3]
```diff
@@ -4,160 +4,391 @@
...
+
+dialog button {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+}
+
+dialog button svg {
+  height: 100%;
+  width: 100%;
```
Comment (piitaya) [inline rootfs/usr/share/www/static/styles.css:306]:
```suggestion
  height: 24px;
  width: 24px;
```
Icon size must be `24px`

File: rootfs/usr/share/www/static/styles.css [hunk 4]
```diff
@@ -4,160 +4,391 @@
...
+  padding: 0;
+}
+
+dialog::backdrop {
+  background: rgba(0, 0, 0, .32);
+}
+
+dialog button {
+  width: 50px;
+  height: 50px;
```
Comment (piitaya) [inline rootfs/usr/share/www/static/styles.css:300]:
```suggestion
  width: 48px;
  height: 48px;
  padding: 12px;
  margin: -12px;
```

**Review by piitaya — COMMENTED** [review]
Not important but the HA logo is not the same as onboarding.

File: rootfs/usr/share/www/index.html (hunk 2 above)
Comment (bramkragten) [reply rootfs/usr/share/www/index.html:118]: Where do you want this padding exactly?

Comment (piitaya) [reply rootfs/usr/share/www/index.html]: `mwc-list` has a 8px padding. I would wrap the link in a div and add `padding: 8px 0` to this div.

**Comment by agners** [comment]
Cool, nice to see this comes alive! Download App Looks weird on Firefox (117) here:

image

**Review by piitaya — APPROVED** [review]

**Review by agners — APPROVED** [review]

**Review by agners — APPROVED** [review]

**Pull request merged**